lib3h/Cargo.toml

```diff
@@ -8,3 +8,6 @@ edition = "2018"
 holochain_logging = { path = "../logging" }
 holochain_lib3h_protocol = { path = "../lib3h_protocol" }
 holochain_sodium = { path = "../sodium" }
+serde = "=1.0.89"
+serde_derive = "=1.0.89"
+serde_json = { version = "=1.0.39", features = ["preserve_order"] }
```

Comment:
I think it'd be awesome if we could remove any json assumptions from lib3h. lib3h itself doesn't need to serialize to json as it'll just be receiving the enums directly.

Reply:
I like banning serde (json). Felt like needed it for handling network config but doing it manually in net crate is not hard,